packages/solana/lib/src/subscription_client/subscription_client.dart

```diff
@@ -32,6 +30,20 @@ class SubscriptionClient {
   /// Subscribe to all incoming messages.
   Stream<SubscriptionMessage> allMessages() => _stream.map(_parse);
 
+  /// Waits for transation with [signature] to reach [status].
```

Comment:
Why not keep it as an extension? I'd like the classes to resemble their actual equivalents in the API.

Reply:
Yeah, valid point. I'll revert it back to extension.